content/guides/monitors.md

```diff
@@ -49,15 +49,47 @@ on the left. This guide will walk through the configuration of the Metric type.
 
 ### Choose what to monitor
 
-1. Select the metric and scope you want to monitor.
+1. Select the alert type.
+    {{< img src="monitor/alert_type.png" alt="alert type" >}}
+
+    A **threshold alert** will compare the value in the selected
+    timeframe against a given threshold. There are additional options available
+    in the alerting conditions section. This is the standard alert case
+    where you know what sort values are unexpected.
+
+    A **change alert** will compare the change or % change of a value between
+    now and some time ago against a given threshold.
+    The compared data points will not be single points but are computed using
+    the parameters in the *alert conditions* section.
+
+    This type of alert is useful to track fast spikes or drops as well as slow
+    changes in a metric when you might not have an exact "unexpected" threshold.
+    *Note:* the calculated value is not the absolute value - meaning it will be
+    negative for a downward change.
+
+    **Anomaly Detection** is an algorithmic feature that allows you to identify
+    when a metric is behaving differently than it has in the past, taking into
+    account trends, seasonal day-of-week and time-of-day patterns. It is well-
+    suited for metrics with strong trends and recurring patterns that are hard
+    or impossible to monitor with threshold-based alerting.
+
+    **Outlier Detection** is an algorithmic feature that allows you to detect
+    when some members of a group are behaving strangely compared to the others.
+    For example, you could detect that one web server in a pool is processing an
+    unusual number of requests, and hence should be a target for replacement. Or,
+    you could get an early warning that significantly more 500s are happening in
+    one AWS Availability Zone (AZ) than the others, which might indicate an issue
+    brewing in that AZ.
+
+2. Select the metric and scope you want to monitor.
   {{< img src="monitor/metric_scope.png" alt="metric scope" >}}
 
     You can create a monitor on any metrics that you are currently sending to
     Datadog. The standard scoping rules apply here. Please refer to the
-    [scope section](/graphingjson/#scope) of the Graphing Primer using JSON for
+    [scope section](/graphing/#scope) of the graphing primer for
```

Comment:
omit the last sentence (the one beginning "Please") and make previous sentence:

"The standard [scoping rules](/graphing/#scope) apply here."